### PR TITLE
feat: Add support for hosting an artifacthub-repo.yml file

### PR DESCRIPTION
**Pull request by cbuto**
Fixes https://github.com/helm/chartmuseum/issues/419.

This is an initial take at hosting an `artifacthub-repo.yml` from ChartMuseum.

Currently, it will only support setting the `repositoryID` in the `artifacthub-repo.yml` file, but adding more fields in the future shouldn't be too difficult.

Let me know what you think, especially the CLI flag implementation for multitenancy.

### CODE REVIEW

File: cmd/chartmuseum/main.go [hunk 1]
```diff
@@ -116,6 +117,7 @@ func cliHandler(c *cli.Context) {
 		Host:                   conf.GetString("listen.host"),
 		PerChartLimit:          conf.GetInt("per-chart-limit"),
 		WebTemplatePath:        conf.GetString("web-template-path"),
+		ArtifactHubRepoID:      conf.GetStringMapString("artifact-hub-repo-id"),
```
Comment (scbizu) [inline cmd/chartmuseum/main.go:120]: will it be nil if no flag set ? Will be panic if no `artifact-hub-repo-id` flag set but still visit the `/artifact-repo.yaml` ?

**Comment by scbizu** [comment]
For me , I can accept the flag set mechanism for multi-tenancy but it maybe hard to set those flag if users have a large set of tenancy , any thoughts about the stdin (i.e.: json file stdin) ?

**Comment by scbizu** [comment]
>  Maybe supporting a combination of the two could be implemented

Yeah , I mean that .

**Review by jdolitsky — APPROVED** [review]
looks great!

**Review by scbizu — APPROVED** [review]

**Pull request merged**